src/robot/libraries/BuiltIn.py

```diff
@@ -2944,7 +2944,7 @@ def _yield_logged_messages(self, messages):
             else:
                 yield value
 
-    def log_to_console(self, message, stream='STDOUT', no_newline=False):
+    def log_to_console(self, message, stream='STDOUT', no_newline=False, format = ""):
```

Comment:
Please change this to `format=''`. Spaces around `=` are against PEP-8 and we have generally used `'` instead of `"` in our code base. The latter isn't too big a deal, but using different quotes in arguments (there's `'STDOUT'` already) would be somewhat inconsistent.